### ISSUE TITLE
Can I combine charts from two projects?

### ISSUE THREAD

**Issue by kevinlee325** [wandb/local#63]
I am working on an object detection task and trying two different models (faster-rcnn and mask-rcnn). For each model, l created a project and logged the validation loss.
I want to compare the two loss charts. First, I open the project of faster-rcnn and remember the min loss. Then, I open the project of mask-rcnn and compare the loss. Can I combine the loss charts into one chart?
The reason I did not just create one project is that for each model I want to change the hyper-parameters and compare different runs.
Thanks for any help and suggestion.
